src/views/admin/PersonCreationView.vue

```diff
@@ -251,6 +353,69 @@
     router.push({ name: 'create-person' });
   };
 
+  // Calculates the next 31st of July (End of school year)
+  // Time here is in german iso format but will later be converted to UCT
+  function getNextJuly31st(): string {
```

Comment:
```suggestion
  function getNextSchuljahresende(): string {
```
Muss nicht sein, klingt nur schöner und ist allgemeiner, falls sich das Datum mal ändert.

Reply:
Stimmt